packrat/src/main/java/com/github/reflectoring/infiniboard/packrat/widget/WidgetConfigRepository.java

```diff
@@ -11,4 +13,6 @@
 
     List<WidgetConfig> findAllByLastModifiedAfter(LocalDateTime date);
 
+    Page<WidgetConfig> findAll(Pageable pageable);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)